Content.Server/_DV/CartridgeLoader/Cartridges/NanoChatCartridgeSystem.cs

```diff
@@ -455,6 +464,22 @@ private void UpdateUIForCard(EntityUid cardUid)
         }
     }
 
+    /// <summary>
+    ///     Updates the UI for all PDAs containing a NanoChat cartridge.
+    /// </summary>
+    private void UpdateUIForAllCards()
+    {
+        // Find any PDA containing this card and update its UI
+        var query = EntityQueryEnumerator<NanoChatCartridgeComponent, CartridgeComponent>();
+        while (query.MoveNext(out var uid, out var comp, out var cartridge))
+        {
+            if (cartridge.LoaderUid == null)
+                continue;
+
+            UpdateUI((uid, comp), cartridge.LoaderUid.Value);
```

Comment:
use `if (cartridge.LoaderUid is {} loader)` and pass loader instead of cartridge.LoaderUid.Value